rand_core/src/impls.rs

```diff
@@ -125,6 +131,7 @@ fn fill_via_chunks<T: Observable>(src: &mut [T], dest: &mut [u8]) -> (usize, usi
 /// }
 /// ```
 pub fn fill_via_u32_chunks(src: &mut [u32], dest: &mut [u8]) -> (usize, usize) {
+    // TODO(SemVer): src: `&[u32]` as we don't mutate it.
     fill_via_chunks(src, dest)
 }
```

Comment:
Does anyone know why this method is public at all? It is used in the impl of `RngCore for BlockRng<R: BlockRngCore>`, which is crate-local. I doubt RNGs would want to use it directly. It looks like this `pub fn` pre-dates `BlockRng`.

I suggest that we deprecate these two methods.

Reply:
I was wondering that myself

Reply:
My guess is that it was intended to be used by other crates implementing `RngCore`.